source/library/bank_helpers.h

```diff
@@ -1,19 +1,19 @@
 // Contains functions to help with working with multle PRG/CHR banks
 
-// Maximum level of recursion to allow with banked_call and similar functions. 
```

Comment:
My editor automatically trims white space when I save a file - I can undo these changes if they're undesired

Reply:
No need; it's fine as-is! If the changes made it harder to tell what was actually done here I might request that, but this seems perfectly clear to me.